### ISSUE TITLE
[Slider] Flaky test

### ISSUE THREAD

**Issue by jverkoey** [material-components/material-components-ios#3203]
```
components/Slider/tests/unit/SliderTests.m:354: error: -[SliderTests testAccessibilityValue] : (([self.slider accessibilityValue]) equal to (expected)) failed: ("30%") is not equal to ("29%")
```

Detected in #3201.

---
https://www.pivotaltracker.com/story/show/156666601

**Comment by romoore** [comment]
Detected in #3159 as well.

**Comment by jverkoey** [comment]
Ran into this again in https://github.com/material-components/material-components-ios/pull/3313.

**Comment by jverkoey** [comment]
Again in https://fusion.corp.google.com/runanalysis/test/prod%3AMaterialComponents_iOS%2Fmacos_external%2Fpresubmit_bazel_xcode_920/prod%3AMaterialComponents_iOS%2Fmacos_external%2Fpresubmit_bazel_xcode_920/KOKORO/b29410a3-5122-4277-a9e9-6687c97fe546/1523566860264/prod%3AMaterialComponents_iOS%2Fmacos_external%2Fpresubmit_bazel_xcode_920%20Build%20%23194/Target%20Transitions?target=MaterialComponents_iOS/macos_external/presubmit_bazel_xcode_920:

```
Test Case '-[SliderTests testAccessibilityValue]' started.
components/Slider/tests/unit/SliderTests.m:1000: error: -[SliderTests testAccessibilityValue] : (([self.slider accessibilityValue]) equal to (expected)) failed: ("30%") is not equal to ("29%")
```
